### PR TITLE
Update muon 8.0.6

### PR DESCRIPTION
**Pull request by bsclifton**
Besides updating to Muon 8.0.6, this also fixes a dependency which was breaking Linux builds

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -107,6 +107,7 @@
     "file-loader": "0.11.2",
     "font-awesome": "^4.5.0",
     "fs-extra": "^2.1.2",
+    "home-path": "^1.0.6",
```
Comment (diracdeltas) [inline package.json]: is this suppoed to be here?

Comment (diracdeltas) [reply package.json]: oh i see, was it breaking linux builds?

seems like there should be a package-lock.json change for it

Comment (bsclifton) [reply package.json]: @diracdeltas yes - I believe it's a dependency of `electron-download`. Without this, the npm install fails on Linux. I'm not exactly sure where this broke, but you can see more details at https://jenkins.brave.com/view/laptop%20builds%20(child%20jobs)/job/browser-laptop-build-linux/536/console

Comment (bsclifton) [reply package.json]: Let me update the package-lock.json too...

Comment (bsclifton) [reply package.json]: @diracdeltas updated 😄 👍

**Comment by DUDENICK** [comment]
lectron-download

**Comment by bsclifton** [comment]
master 3a39fe728040b756f8bef441dc2f64d89e318a33
0.24.x a27f523ce91cb696ac56e5ac67fe75551165f497
0.23.x 41428f9fbe0e9ce41bae6e0625bc326c0e79b2fd

**Pull request merged**